src/args.ts

```diff
@@ -37,12 +30,15 @@ const handleNoCommand = async () => {
 export const getYargs = async (): Promise<ReviewArgs> => {
   const argv = yargs
     .option("ci", {
-      description: "Indicate that the script is running on a CI environment",
-      type: "boolean",
-      default: false,
+      description:
+        "Indicates that the script is running on a CI environment. Specify which platform the scrip is running on, 'github' or 'gitlab'.",
+      choices: ["github", "gitlab"],
+      type: "string",
+      default: "github",
```

Comment:
This will break local running of the script. Consider having default as local or something of that nature.

Reply:
Hmm okay! I can see the use in protecting it when running locally. But theoretically we shouldn't be running it locally with a --ci flag right?

Reply:
Have you tested the output when you don't add --ci? Is it undefined or GitHub? 

I'm guessing it will default to GitHub but I am just guessing. 😀